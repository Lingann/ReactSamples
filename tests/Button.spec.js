import React from 'react';
import { create } from "react-test-renderer";

const Button = () => {
    return <button> Nothoing to do for now </button>
}

describe("Button component", ()=> {
    test("Match the snapshoot", () => {
        const  button = create(<Button />);
        expect(button.toJSON()).toMatchSnapshot();
    })
})
