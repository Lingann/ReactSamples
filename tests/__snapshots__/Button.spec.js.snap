// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Match the snapshoot 1`] = `
<button>
   Nothoing to do for now 
</button>
`;
